### DISCUSSION TITLE
Disable/hide temporary option of user password reset form

### DISCUSSION THREAD

**Discussion by sergiotanaka** [keycloak/keycloak#33755] [Q&A] [answered]
Hello everyone,

A customer asked if it would be possible to disable/hide the temporary option on the user's password reset form.
Is there any way to configure this in current versions of Keycloak? (version I'm working on: 25.0.4 / Admin theme: keycloak.v2).
Note: default value of this field should be true.

![image](https://github.com/user-attachments/assets/55cf938e-d807-4526-a208-a61967d26a60)

Thanks

**Comment by jonkoops** [comment, marked as answer]
This is currently not possible. If you'd like to have this as an enhancement I would propose you log an enhancement request in the issue tracker. Please make sure to not just describe the enhancement, but also the motivation as to why you'd want to have this implemented in this manner.
